<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OthelloClass</class>
 <widget class="QMainWindow" name="OthelloClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>170</width>
      <height>190</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Pion</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Noir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Blanc</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Position :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rang√©es</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbRangees"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Colonnes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbColonnes"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlacer">
       <property name="text">
        <string>Placer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>210</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Othello">
    <property name="title">
     <string>&amp;Othello</string>
    </property>
    <addaction name="action_Nouveau"/>
    <addaction name="action_Fermer"/>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Observateur">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Observateur</string>
    </property>
    <addaction name="action_Texte"/>
    <addaction name="action_Expert"/>
   </widget>
   <widget class="QMenu" name="menu_A_Propos">
    <property name="title">
     <string>&amp;A propos</string>
    </property>
    <addaction name="actionA_ide"/>
    <addaction name="actionA_uteur"/>
   </widget>
   <addaction name="menu_Othello"/>
   <addaction name="menu_Observateur"/>
   <addaction name="menu_A_Propos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Nouveau">
   <property name="text">
    <string>&amp;Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Fermer">
   <property name="text">
    <string>&amp;Fermer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Texte">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Texte</string>
   </property>
  </action>
  <action name="action_Expert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Expert</string>
   </property>
  </action>
  <action name="actionA_uteur">
   <property name="text">
    <string>A&amp;uteur</string>
   </property>
  </action>
  <action name="actionA_ide">
   <property name="text">
    <string>A&amp;ide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
